src/System.Private.Uri/src/System/UriExt.cs

```diff
@@ -385,12 +386,12 @@ public bool IsWellFormedOriginalString()
 
         public static bool IsWellFormedUriString(string uriString, UriKind uriKind)
         {
```

Comment:
Should we accept a null `uriString` here? It will return false.

Reply:
Yup. Fixed.